### PR TITLE
As a member of the community, I want these Profile Privacy Features, so I can be more private

### PR DESCRIPTION
**Pull request by owocki**
##### Description

As a member of the community, I want these Profile Privacy Features, so I can be more private

1. set the default profile hidden = False, but profile visible = True when the user logs in
1. write a migration such that anyone who's never logged into gitcoin before will not have a profile unless they explicity sign up.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
profiles

##### Refers/Fixes
https://github.com/gitcoinco/web/issues/1091

### CODE REVIEW

File: app/dashboard/migrations/0069_auto_20180507_1841.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.0.4 on 2018-05-07 18:41
+
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    Profile = apps.get_model('dashboard', 'Profile')
+    profiles = Profile.objects.filter(github_access_token='').all()
```
Comment (mbeacom) [inline app/dashboard/migrations/0069_auto_20180507_1841.py]: We'll want to make sure we're not marking valid profiles to hidden if they have a `user` as well.  Also, we should be able to just do a `Profiles.objects.filter(github_access_token='').update(hide_profile=True)`

**Comment by mbeacom** [comment]
Also, looks like we'll need to update the Leaderboard tests.

File: app/dashboard/migrations/0069_auto_20180507_1841.py (hunk 1 above)
Comment (owocki) [reply app/dashboard/migrations/0069_auto_20180507_1841.py]: 3d7e03ad8996c3057490b56d8846313f09af67a8

**Comment by owocki** [comment]
> Also, looks like we'll need to update the Leaderboard tests.

yes just updated

**Pull request merged**